ui/components/component-library/avatar-account/avatar-account.types.ts

```diff
@@ -0,0 +1,22 @@
+import { Size } from '../../../helpers/constants/design-system';
+
+export enum AvatarAccountVariant {
+  Jazzicon = 'jazzicon',
```

Comment:
Can u use `jazzicon` and `blockies` as camel case?

Reply:
So a few of our string value enums are used for dynamic classNames or will be once they are converted to enums e.g.
```
className={classNames(`mm-avatar-account--size-${size}`}
```

 So I thought I would update it here too. In this case for variants they are not so it would be fine to update to PascalCase. But is there a benefit to matching the value to the key?

Reply:
Oh i actually mean the name of the key, i would defo keep the value as it is, but to change the key to camel:
```
export enum AvatarAccountVariant {
  jazzicon = 'jazzicon',
  }
```
So when it is used in classNames we don't need to change anything and still use `jazzicon` as it was.
Hope it makes sense ！

Reply:
After discussion will use `Pascal case` as our conversion  ~